src/clib/pio_file.c

```diff
@@ -109,6 +109,7 @@ int PIOc_createfile(const int iosysid, int *ncidp, int *iotype,
         return PIO_ENOMEM;
 
     /* Fill in some file values. */
+    file->fh = -1;
```

Comment:
Yes it does help in debugging to set this to a known value on tasks where it is not used.